tests/test_ioc_system.py

```diff
@@ -603,6 +603,34 @@ async def test_metadata_parses_into_multiple_pvs_caput_single_pv(
     ) in multiprocessing_queue_to_list(command_queue)
 
 
+async def test_non_defined_seq_table_can_be_added_to(
```

Comment:
I think it's probably worth writing a second system test, that has an empty SEQ3 and then tries to add data via writing PVs. Currently this only tests the flow in one direction.

Reply:
As discussed over slack, we'll handle this in a new PR.